tests/unit_tests/test_matchmaker_algorithm_team_matchmaker.py

```diff
@@ -224,16 +226,16 @@ def test_maximum_game_quality_for_even_teams(player):
     search = Search([player])
     team_a = CombinedSearch(*[search] * 4)
     team_b = CombinedSearch(*[search] * 4)
-    game = TeamMatchMaker().assign_game_quality((team_a, team_b), 4)
+    game = TeamMatchMaker().assign_game_quality((team_a, team_b), 4, 1000)
 
-    assert game.quality == 1.0
+    assert game.quality >= 1.0
```

Comment:
Game quality can be better than 1 now?

Reply:
Yes, because of the minority bonus. I set the minority bonus to 0 in the test above, which checks that the maximum game quality without bonuses is still 1. This test here also assures that the miority bonus is only positive.